### DISCUSSION TITLE
Remove Stripe account from GitHub sponsors

### DISCUSSION THREAD

**Discussion by menkotoglou** [community/community#30709] [Other Feature Feedback, Questions, & Ideas] [answered]
Regarding my GitHub sponsors account connected with Stripe, there seem to be 3 of them connected to my profile. The first two seem to have been created by accident, so I'd like them to be deleted so that I proceed further with my GitHub sponsors account completion. I chatted with Stripe support, and they told me that only GitHub would be able to delete them, since they're the account managers, by just making an API call providing only the account id. You can find the relevant documentation here: https://stripe.com/docs/api/accounts/delete.

How can I reach out from someone from the support to assist with this?

Thanks everyone in advance!

**Comment by KasperiP** [comment, marked as answer]
Open support ticket [here](https://support.github.com/request). I had issues with GitHub Sponsors earlier and after sending a support request it took 1-2 days to get answered and my issue got fixed.

Choose account or billing issues => and from the "What do you need help with?"-dropdown select GitHub sponsors. Then just explain your issue in the text field.

**Reply by TheAnkushshah** [reply]
I started using GitHub sponsors now I accidently added 3 stripe connect/express accounts, and when I'm trying to delete, it pops up this error stating that:
(Your stripe connect account could not be deleted. Please try again later or contact support.)

This is the sponsor account URL:
(https://github.com/sponsors/HavenFutures)

Email connected to that account is (support@havenfutures.com)

Due to this, I unpublished and disabled my sponsor account expecting that it would also delete stripe connect/express accounts but didn't, so I request you guys to make an API call and delete all 3 stripe connect/express accounts.

**Comment by emploiyer** [comment]
il faut envoyer identité de ces deux compte donc tu veux supprimer

**Reply by TheAnkushshah** [reply]
I started using GitHub sponsors now I accidently added 3 stripe connect/express accounts, and when I'm trying to delete, it pops up this error stating that:
(Your stripe connect account could not be deleted. Please try again later or contact support.)

This is the sponsor account URL:
(https://github.com/sponsors/HavenFutures)

Email connected to that account is ([support@havenfutures.com](mailto:support@havenfutures.com))

Due to this, I unpublished and disabled my sponsor account expecting that it would also delete stripe connect/express accounts but didn't, so I request you guys to make an API call and delete all 3 stripe connect/express accounts.

**Comment by jenlampton** [comment]
I just wanted to post an update here that we can delete the Stripe accounts ourselves now.  

If you can get to `https://github.com/sponsors/USERNAME/dashboard/settings` then there is a section labeled `All Stripe accounts` and each has a little bin/trash icon we can use to delete!

**Reply by maomincoding** [reply]
Hi, I tried your method but got this error.
![image](https://github.com/user-attachments/assets/d9175501-99e8-47e7-baef-6900bc48df6a)

**Reply by TheAnkushshah** [reply]
I started using GitHub sponsors now I accidently added 3 stripe connect/express accounts, and when I'm trying to delete, it pops up this error stating that:
(Your stripe connect account could not be deleted. Please try again later or contact support.)

This is the sponsor account URL:
(https://github.com/sponsors/HavenFutures)

Email connected to that account is (support@havenfutures.com)

Due to this, I unpublished and disabled my sponsor account expecting that it would also delete stripe connect/express accounts but didn't, so I request you guys to make an API call and delete all 3 stripe connect/express accounts.
